contingency/contingency-api/src/test/java/com/powsybl/contingency/ContingencyTest.java

```diff
@@ -154,4 +161,59 @@ public void validationTestForTwt3() {
                         .map(Contingency::getId)
                         .collect(Collectors.toList()));
     }
+
+    @Test
+    public void testIdentifiableBuilder() {
```

Comment:
could you test maybe a contingency with 2 or more elements created with this new method